content/courses/native-onchain-development/deserialize-instruction-data.md

```diff
@@ -599,8 +599,7 @@ address, you can generate a new program ID from the playground before deploying.
 You can test your program by submitting a transaction with the right instruction
 data. For that, feel free to use
 [this script](https://github.com/Unboxed-Software/solana-movie-client) or
-[the frontend](https://github.com/Unboxed-Software/solana-movie-frontend) we
-built in the
+[the frontend](https://github.com/EmekaManuel/movie-review-dapp) we built in the
```

Comment:
Update this link to use solana-developer/movie-review-frontend

Reply:
> Update this link to use solana-developer/movie-review-frontend

updated ✅